### PR TITLE
Remove 0 add 1

### PR DESCRIPTION
**Pull request by wcampbell0x2a**

### CODE REVIEW

File: src/game/piece.rs [hunk 1]
```diff
@@ -168,10 +168,10 @@ impl Piece {
...
         } else if r#move == Movement::Down {
             return [
-                (self.positions[0].0 + 1, self.positions[0].1),
-                (self.positions[1].0 + 1, self.positions[1].1),
-                (self.positions[2].0 + 1, self.positions[2].1),
-                (self.positions[3].0 + 1, self.positions[3].1),
+                (self.positions[0].1, self.positions[0].1),
+                (self.positions[1].1, self.positions[1].1),
+                (self.positions[2].1, self.positions[2].1),
+                (self.positions[3].1, self.positions[3].1),
```
Comment (Brian-J-Billman) [inline src/game/piece.rs:174]: hey uhhhhhh, I don't think this works quite like you were thinking it would

Comment (Brian-J-Billman) [reply src/game/piece.rs:174]: I'm guessing you just used sed -i lol

Comment (wcampbell0x2a) [reply src/game/piece.rs:174]: git ls-files, uhhh whoops

Comment (Brian-J-Billman) [reply src/game/piece.rs:174]: you should checkout this branch and try to play the game with it. It's pretty cool xd

**Pull request merged**